Siutan's HW6

siutanwong opened the pull request.

Commit:
Uploding our first notes

Commit:
my changes

danilnagy commented:
Hi Siu Tan, it looks like the file got somehow messed up by Github. Could you take a look? Also, have you implemented any of the features of the assignment?

siutanwong committed:
update

siutanwong committed:
change

siutanwong commented:
Hi Danil,
Thanks for pointing out the problem. I've fixed the files and updated them
on my Github: https://github.com/siutanwong/week-6

Thanks,
Siutan

On Tue, Nov 10, 2015 at 12:13 PM, danilnagy notifications@github.com
wrote:

> Hi Siu Tan, it looks like the file got somehow messed up by Github. Could
> you take a look? Also, have you implemented any of the features of the
> assignment?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/data-mining-the-city/week-6/pull/5#issuecomment-155491527
> .
